src/ngraph/op/op.hpp

```diff
@@ -40,10 +40,9 @@ namespace ngraph
 
             virtual bool is_op() const override { return true; }
         protected:
-            Op(const std::string& node_type, const NodeVector& arguments);
-
-        private:
             std::shared_ptr<ngraph::op::util::OpAnnotations> m_op_annotations;
```

Comment:
I don't think there is any need to move this to `protected`. We have `get_op_annotations()`, and can `clone` from that if we need to.

Reply:
Ok, I can do that instead but I'm not sure I understand the rationale. Why should we hide `m_op_annotations` in the subclass and use a public interface if exposing it it's not compromising the correctness of the implementation?

Reply:
My first instinct is always to keep data members private, except in extremely limited circumstances, on the theory that it provides better separation of implementation and interface.